connectors/sources/salesforce.py

```diff
@@ -1340,10 +1356,36 @@ def get_default_configuration(cls):
                 "tooltip": "The client secret for your OAuth2-enabled connected app. Also called 'consumer secret'",
                 "type": "str",
             },
+            "standard_objects_to_sync": {
+                "display": "textarea",
+                "label": "Standard Objects to Sync",
+                "order": 4,
+                "tooltip": "The list of Salesforce Standard Objects to sync. Remove any from list to exclude from sync.",
+                "type": "list",
+                "value": ", ".join(STANDARD_SOBJECTS),
+                "required": False,
+            },
+            "sync_custom_objects": {
+                "display": "toggle",
+                "label": "Sync Custom Objects",
+                "order": 5,
+                "tooltip": "Whether or not to sync custom objects",
+                "type": "bool",
+                "value": False,
+            },
+            "custom_objects_to_sync": {
+                "display": "textarea",
+                "label": "Sync Custom Objects",
```

Comment:
To be consistent with `"label": "Standard Objects to Sync",`
```suggestion
                "label": "Custom Objects to Sync",
```